src/components/Contributing/index.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import styles from "./styles.module.css";
+
+const contributing = [

```

Comment:
We'll also need alt text here, I think.

Reply:
The `title` variable is the alt text for the image. I will create a separate variable for `alt` for better visibility.